src/coreclr/tools/aot/ILCompiler.ReadyToRun/ILCompiler.ReadyToRun.csproj

```diff
@@ -102,6 +102,7 @@
     <Compile Include="..\..\Common\Compiler\NameMangler.cs" Link="Compiler\NameMangler.cs" />
     <Compile Include="..\..\Common\Compiler\SingleMethodRootProvider.cs" Link="Compiler\SingleMethodRootProvider.cs" />
     <Compile Include="..\..\Common\Compiler\PropertyPseudoDesc.cs" Link="Compiler\PropertyPseudoDesc.cs" />
+    <Compile Include="..\..\Common\Compiler\EventPseudoDesc.cs" Link="Compiler\EventPseudoDesc.cs" />
```

Comment:
I missed when PropertyPseudoDesc was being added to crossgen2. This and EvenPseudoDesc are just NativeAOT warts. I'd still like to do something about them at some point. It was a quick hack when porting the dataflow anaysis originally. We should not be importing it to crossgen2. Could you instead move the helper from /Common/Compiler/MethodExtensions.cs to a place that is specific to ILCompiler.Compiler? Can be PseudoDescExtensions.cs or something like that.

Reply:
I think I made all the necessary changes to make it work PTAL